app/views/CartPage/index.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { useCallback, useState } from 'react';
+import { Button, Container, ListView, Message, NumberInput, TextOutput } from '@the-deep/deep-ui';
+import {
+    gql,
+    useMutation,
+    useQuery,
+} from '@apollo/client';
+import { AiTwotoneDelete } from 'react-icons/ai';
+import { FaHeart } from 'react-icons/fa';
```

Comment:
If possible use io5, if not in io5 use io4. Under drastic circumstances use other libraries